lib/core/init.js

```diff
@@ -25,6 +25,9 @@ let start;
 export function init() {
 	if (location.hash === RES_DISABLED_HASH) return;
 
+	// Only open RES in iframes that are reddit embedded pages.
+	if (window.location !== window.parent.location && !(window.location.search.includes('embedded=true') || window.location.pathname === '/options.html')) return;
```

Comment:
I've moved the init logic to foreground.entry. https://github.com/honestbleeps/Reddit-Enhancement-Suite/commit/ee0c18d06db840fcd3562bd731ad155f48036e3e

`(new URL(location.href)).searchParams.get('embedded') === 'true'` would be more accurate than `window.location.search.includes('embedded`

Reply:
Fair enough, I have changed it to follow that. Also because I feel that it doesn't need to throw an error (it not loading on other iframes is the goal) I wrapped `init()` in the check. 



Do you have a suggestion for the css issue? I could just use 

```css 
.res .blueButton
```

To make it more specific but wanted to check that first.

Reply:
Yeah, throwing errors is a bit too much. I've tweaked the init code in 968914f484b1f6cf2fda0259c319200142557528.

> `window.location !== window.parent.location`

Any reason to check this way instead of simply `window !== window.parent`?

> Do you have a suggestion for the css issue?

Probably simplest to just add `!important` to the `color` rule.

Reply:
> Any reason to check this way instead of simply window !== window.parent?

It did seem like a more concrete check but other than that I don't really have a reason. 

> Probably simplest to just add !important to the color rule.

Alright, will do.

Reply:
Alright, all fixed.